ansible/roles/ocp-workload-fsi-client-onboarding-demo/tasks/workload.yml

```diff
@@ -19,10 +19,18 @@
 #  shell: "oc create -f /tmp/{{guid}}//limit-range.yaml -n {{ocp_project}}"
 
 - name: Import ImageStreams
-  shell: "oc create -f https://raw.githubusercontent.com/jboss-openshift/application-templates/master/processserver/processserver64-image-stream.json -n {{ocp_project}}"
+  shell: "oc create -f https://raw.githubusercontent.com/jboss-container-images/jboss-processserver-6-openshift-image/6.4.x/templates/processserver64-image-stream.json -n {{ocp_project}}"
+
+- name: Patch Image Stream
+  shell: |
+        for i in {0..5}
+        do
+          JSONPATCH='[{"op": "replace", "path": "/spec/tags/'$i'/from/name", "value": "registry.access.redhat.com/jboss-processserver-6/processserver64-openshift:1.'$i'"}]'
+          oc patch is/jboss-processserver64-openshift --type='json' -p "$JSONPATCH"
+        done
```

Comment:
Don't you want to use an ansible loop here instead ?

I think it's very unlikely, but what if one of the `oc patch` fails for example with `i=3` ? the task will still succeed.

Reply:
Have added a commit which replaces the for-loop with an Ansible Loop.